tests/devtools.test.tsx

```diff
@@ -437,6 +439,8 @@ describe('when it receives an message of type...', () => {
 })
 
 it('works with redux middleware', () => {
```

Comment:
Here's the trick. Not super elegant, and hope to find a better way.
```suggestion
it('[DEV-ONLY] works with redux middleware', () => {
```